### PR TITLE
Solved challenge

### PR DESCRIPTION
**Pull request by svetlivaksevski**

### CODE REVIEW

File: css-grid_place-elements/css/styles.css [hunk 1]
```diff
@@ -68,24 +68,38 @@ body {
 
 .element-1 {
   /* Add styles here */
+  grid-column-start: 3;
+  grid-column-end: 5;
+
```
Comment (henriquepalazzo) [inline css-grid_place-elements/css/styles.css:73]: You can use the short version grid-column: 3/5

File: css-grid_place-elements/css/styles.css [hunk 2]
```diff
@@ -68,24 +68,38 @@ body {
...
 }
 
 .element-2 {
   /* Add styles here */
+  grid-column-start: 5; 
+  grid-column-end: 11;
 }
```
Comment (henriquepalazzo) [inline css-grid_place-elements/css/styles.css:80]: When the element goes until the end of the grid, you can use -1, so you don't have to count: grid-column: 5/-1.

File: css-grid_place-elements/css/styles.css [hunk 3]
```diff
@@ -68,24 +68,38 @@ body {
...
 
 .element-5 {
   /* Add styles here */
+  grid-column-start: 3;
+  grid-column-end: 5;
+  grid-row: span 2;
```
Comment (henriquepalazzo) [inline css-grid_place-elements/css/styles.css:86]: That's a nice idea, better than mine, I explicitly defined the start and end, your solution is cleaner.

**Comment by henriquepalazzo** [comment]
All good Svetli! I commented a few points on your code, as a suggestion of improvement.

File: css-grid_place-elements/css/styles.css (hunk 2 above)
Comment (svetlivaksevski) [reply css-grid_place-elements/css/styles.css:80]: Very nice tip. Thank you. WIll use it!

**Pull request merged**